application/src/main/resources/thingsboard.yml

```diff
@@ -930,7 +930,7 @@ edges:
   storage:
     max_read_records_count: "${EDGES_STORAGE_MAX_READ_RECORDS_COUNT:50}"
     no_read_records_sleep: "${EDGES_NO_READ_RECORDS_SLEEP:1000}"
-    sleep_between_batches: "${EDGES_SLEEP_BETWEEN_BATCHES:1000}"
+    sleep_between_batches: "${EDGES_SLEEP_BETWEEN_BATCHES:10000}"
```

Comment:
Why this parameter is changed?

Reply:
This is a timeout, that historically was set incorrectly from the begging.
1 second is not enough in case of a slow internet connection to deliver all messages in a batch, and because of this new process of resending the batch started. And because of started reprocessing, some of the messages, that are successfully delivered, are pushed to the edge again, which causes duplicates of the message and useless traffic data.
The New 10-second interval is giving more chances to deliver messages in a slow connection case without duplicates.
It doesn't mean 10 seconds pause - as soon as all messages in the batch are successfully delivered, the new batch process started.